src/runner/browser-env/vite/browser-modules/globals.ts

```diff
@@ -9,7 +9,6 @@ const connectToSocket = (): BrowserViteSocket => {
             runUuid: window.__testplane__.runUuid,
             type: BROWSER_EVENT_PREFIX,
         },
-        transports: ["websocket"],
```

Comment:
Do not specify transports explicitly because user tunneler may not support ws connection. More info about transfport in socketio - https://socket.io/docs/v4/engine-io-protocol/#transports